plugins/modules/cloud/univention/udm_dns_record.py

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/python
+# -*- coding: utf-8 -*-
 # -*- coding: UTF-8 -*-
```

Comment:
```suggestion

```

Reply:
Hi @zanssa , thanks for reviewing. @felixfontein had pointed that already, and I have just pushed the update.

This handful of files *had* the marker with `UTF-8` (uppercase), and I had missed that, hence assuming that all occurrences would ever be lowercase.